test/spawn/index.js

```diff
@@ -29,19 +30,47 @@ tap.test('should restart the server twice', t => {
   });
 });
 
-tap.test('should handle errors', t => {
```

Comment:
Please leave the existing tests as they are.

Reply:
My bad, just removed it by accident.